UWPInjector/source/main.cpp

```diff
@@ -70,8 +71,28 @@ void IterateThreads(ThreadCallback ThreadProc, std::uint32_t ProcessID, void* Da
 	CloseHandle(hSnapShot);
 }
 
-int main()
+int main(int argc, char** argv, char** envp)
 {
+	std::uint32_t ProcessID = 0;
+
+	if (true) {
+		for (int i = 1; i < argc; i++) {
+			std::string arg = argv[i];
+			if ( arg == "-h" ){
```

Comment:
This looks good. [Though ideally we can use `string_view` rather than allocate a whole new `std::string` just for the "==" operator.](https://en.cppreference.com/w/cpp/header/string_view)

Reply:
this has been fixed